build_tools/lint.fish

```diff
@@ -128,26 +128,7 @@ if set -q c_files[1]
         # The stderr to stdout redirection is because oclint, incorrectly writes its final summary
         # counts of the errors detected to stderr. Anyone running this who wants to capture its
         # output will expect those messages to be written to stdout.
-        if test "$kernel_name" = "Darwin"
-            if not test -f compile_commands.json
-                xcodebuild -alltargets >xcodebuild.log
-                oclint-xcodebuild xcodebuild.log >/dev/null
```

Comment:
I don't know if this should go - these magic commands made OCLint actually work when it wouldn't work out of the box.

Reply:
Presumably they'd work with an oclint-generated Xcode project?